default.py

```diff
@@ -195,10 +195,10 @@ def get_params():
     Video.PlayStream(name, url, iconimage, description, subtitles_url)
 
 elif mode == 202:
-    Video.AddAvailableStreamItem(name, url, iconimage, description)
+    Video.AddAvailableStreams(name, url, iconimage, description)
```

Comment:
That doesn't sound right. Mode 202 is supposed to be for catchup streams, not live.